docs/docs/command-line-interface.mdx

```diff
@@ -470,6 +477,13 @@ The validator also checks if you have any stories where different assistant acti
 dialogue history. Conflicts between stories will prevent a model from learning the correct
 pattern for a dialogue.
 
+:::info Searching for the `assistant_id` key introduced in 3.5
+
+The validator will check whether the `assistant_id` key is present in the config file and will exit with an error if this
```

Comment:
> the config file
I'd be more specific and name the file, I assume `config.yml`

Reply:
I'm confused because for me `rasa data validate` validates training data, not the config file. Also, it's not required for `rasa train` so I wouldn't expect to see an error here.

Reply:
> rasa data validate validates training data, not the config file

I asked myself the same question but in the end added the config check to the validation checks because of past feedback of `rasa train` and `rasa data validate` not being aligned in terms of warnings and errors. It's a fair point that since `rasa train` doesn't exit in case of missing identifier key, `rasa data validate` shouldn't do this either. I can change it to raise a warning instead of exiting with an error. Wdyt?

Reply:
yeah I think it would be more consistent (cc @madebymagnolia )